<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statuswindow</class>
 <widget class="QDialog" name="Statuswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="terminateProcess">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminate training process</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="trainStatusInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="finishButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>241</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
